neuralmonkey/runners/beamsearch_runner.py

```diff
@@ -100,3 +100,16 @@ def loss_names(self) -> List[str]:
     @property
     def decoder_data_id(self) -> Optional[str]:
         return None
+
+
+def beam_search_runner_range(output_series: str,
+                             decoder: BeamSearchDecoder,
+                             max_rank: int = 1,
```

Comment:
tohle bych mu nedával default jedna, aby to lidi nepoužívali místo normálního beam search runneru, tak jak je to v tom testu. do testu bych napsal že chci dva nejlepší